### PR TITLE
Update dependency litepicker to v2

### PR DESCRIPTION
**Pull request by etp-bot-mu**
This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [litepicker](https://litepicker.com) ([source](https://togithub.com/wakirin/litepicker)) | dependencies | major | [`^1.5.7` -> `^2.0.0`](https://renovatebot.com/diffs/npm/litepicker/1.5.7/2.0.12) |

---

### Release Notes

<details>
<summary>wakirin/litepicker (litepicker)</summary>

### [`v2.0.12`](https://togithub.com/wakirin/litepicker/compare/2.0.11...2.0.12)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.11...2.0.12)

### [`v2.0.11`](https://togithub.com/wakirin/litepicker/compare/2.0.10...2.0.11)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.10...2.0.11)

### [`v2.0.10`](https://togithub.com/wakirin/litepicker/compare/2.0.9...2.0.10)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.9...2.0.10)

### [`v2.0.9`](https://togithub.com/wakirin/litepicker/compare/2.0.8...2.0.9)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.8...2.0.9)

### [`v2.0.8`](https://togithub.com/wakirin/litepicker/compare/2.0.7...2.0.8)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.7...2.0.8)

### [`v2.0.7`](https://togithub.com/wakirin/litepicker/compare/2.0.6...2.0.7)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.6...2.0.7)

### [`v2.0.6`](https://togithub.com/wakirin/litepicker/compare/2.0.5...2.0.6)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.5...2.0.6)

### [`v2.0.5`](https://togithub.com/wakirin/litepicker/compare/2.0.4...2.0.5)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.4...2.0.5)

### [`v2.0.4`](https://togithub.com/wakirin/litepicker/compare/2.0.3...2.0.4)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.3...2.0.4)

### [`v2.0.3`](https://togithub.com/wakirin/litepicker/compare/2.0.2...2.0.3)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.2...2.0.3)

### [`v2.0.2`](https://togithub.com/wakirin/litepicker/compare/2.0.1...2.0.2)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.1...2.0.2)

### [`v2.0.1`](https://togithub.com/wakirin/litepicker/compare/2.0.0...2.0.1)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/2.0.0...2.0.1)

### [`v2.0.0`](https://togithub.com/wakirin/litepicker/compare/v1.5.7...2.0.0)

[Compare Source](https://togithub.com/wakirin/litepicker/compare/v1.5.7...2.0.0)

</details>

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined), Automerge - At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about this update again.

---

 - [ ] If you want to rebase/retry this PR, check this box

---

This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).

### CODE REVIEW

**Comment by solita-antti-mottonen** [comment]
Päivämäärän vaihtaminen ei toimi. Tarvitaan lisää testejä

**Review by solita-antti-mottonen — CHANGES_REQUESTED** [review]
Silently fails on use

File: e2e-tests/cypress/cypress/e2e/kaytonvalvonta/kaytonvalvonta.cy.js [hunk 1]
```diff
@@ -693,4 +699,36 @@ context('Käytönvalvonta', () => {
     cy.get('[data-cy="-submit"]').click();
     cy.get('.alert').should('not.exist');
   });
+
+  it.only('date picking works', () => {
```
Comment (Juholei) [inline e2e-tests/cypress/cypress/e2e/kaytonvalvonta/kaytonvalvonta.cy.js:703]: only

File: e2e-tests/cypress/cypress/e2e/kaytonvalvonta/kaytonvalvonta.cy.js [hunk 2]
```diff
@@ -693,4 +699,36 @@ context('Käytönvalvonta', () => {
...
+    navigateToKaytonvalvonta();
+    createNewValvonta();
+    navigateToValvontaPage();
+    startValvonta();
+
+    cy.get('[data-cy="toimenpide-type-selection"]')
```
Comment (Juholei) [inline e2e-tests/cypress/cypress/e2e/kaytonvalvonta/kaytonvalvonta.cy.js:709]: Tässä voisi varmaan käyttää meidän cy.selectInSelect-komentoa

Comment (Juholei) [inline e2e-tests/cypress/cypress/e2e/kaytonvalvonta/kaytonvalvonta.cy.js:709]: Tässäkin voisi varmaan käyttää cy.selectInSelectiä

File: e2e-tests/cypress/cypress/e2e/kaytonvalvonta/kaytonvalvonta.cy.js (hunk 1 above)
Comment (solita-antti-mottonen) [reply e2e-tests/cypress/cypress/e2e/kaytonvalvonta/kaytonvalvonta.cy.js:703]: Tuo on kyllä naurettava ominaisuus että ainut tapa ajaa yksi testi on muuttaa koodia

**Review by Juholei — APPROVED** [review]

**Pull request merged**